src/components/helpers/validators/email-address-validator.test.js

```diff
@@ -1,12 +1,24 @@
 import isValidEmail from "./email-address-validator";
 
 describe("test validation email helper", () => {
+  const error = "Invalid email address";
```

Comment:
Give names for `describe` and `it` statement properly according to test name conventions, which I shared with all of you